src/rules/at-import-no-import/index.js

```diff
@@ -0,0 +1,29 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("at-import-no-import");
+
+export const messages = utils.ruleMessages(ruleName, {
+  expected: "@import is disallowed, replace it with @use"
```

Comment:
Could you please change the validation message to match the stylelint format:
https://github.com/stylelint/stylelint/blob/master/docs/user-guide/rules/about.md#violation-messages